### ISSUE TITLE
DST Staging Review: Document content read out by screen readers after upload

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#3084]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** File Input Component
**Staging Review ticket:** [Staging Review: (DS) Converted File Input Component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77552)

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**This is an issue with the:** code
**High-priority:** yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When uploading a PDF (and probably other document formats), the thumbnail displayed is an `object` with the file's content encoded in the `data` attribute. This `object` is accessible to screen readers, which means a screen reader user may accidentally get caught listening to the full content of the file they just uploaded.

I don't think this is likely to be desired behavior --- for instance, sighted users aren't going to review their document text in the tiny thumbnail. The thumbnail is a nice visual confirmation but most users will probably be more interested in the filename more than anything else.

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/61139d7c-97a4-4b24-bd5f-ab1cf06283f9

</details>

## Recommended action
I think you're probably safe to put an `aria-hidden="true"` on the `thumbnail-container` div. Screen reader users can skip over that entirely and go straight to the file info. As a bonus, that also helps avoid any need for unique alt text when files that are uploaded are images.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/3084?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @powellkerry @rmessina1010 @rsmithadhoc
